integrations/aircall-basic/actions/create-user.md

```diff
@@ -0,0 +1,53 @@
+# Create User
+
+## General Information
+
+- **Description:** Creates a user in Aircall.
+- **Version:** 0.0.1
+- **Group:** Others
+- **Scopes:**: _None_
+- **Endpoint Type:** Action
+- **Code:** [🔗](https://github.com/NangoHQ/integration-templates/tree/main/integrations/aircall-basic/actions/create-user.ts)
+
+## Endpoint Reference
+
+### Request Endpoint
+
+- **Path:** `/users`
+- **Method:** `POST`
+
+### Request Query Parameters
+
+_No request parameters_
+
+### Request Body
+
+```json
+{
+  "firstName": "<string>",
+  "lastName": "<string>",
+  "email": "<string>"
+}
+```
+
+### Request Response
+
+```json
+{
+  "id": "<string>",
+  "email": "<string>",
+  "firstName": "<string>",
+  "lastName": "<string>"
+}
+```
+
+## Changelog
+
+- [Script History](https://github.com/NangoHQ/integration-templates/commits/main/integrations/aircall-basic/actions/create-user.ts)
+-- [ReadMe History](https://github.com/NangoHQ/integration-templates/commits/main/integrations/aircall-basic/actions/create-user.md)
```

Comment:
Not sure it's common to say `ReadMe` plus it's very git. I guess the point is to share this to customers no?